### DISCUSSION TITLE
Is it possible to let neovide.exe guide to lunarvim?

### DISCUSSION THREAD

**Discussion by JokerJostar** [neovide/neovide#2266] [General]
i hope it is possible

**Comment by usarialan** [comment]
It's possible, you can check the config file from [here](https://neovide.dev/config-file.html). All you need to do is to set `neovim-bin` value to the path to your LunarVim binary. But for some reason, it does not work with the 0.12.0 release (on macOS at least). It is working with the 0.11.2 version though.

**Comment by K4R7IK** [comment]
@usarialan @JokerJostar Were you guys able to make it work?

Never mind, I was able to launch Neovide with a different set of configurations, which I wanted.
